pkg/operator/ceph/object/user/controller.go

```diff
@@ -493,10 +490,11 @@ func updateStatus(client client.Client, name types.NamespacedName, status string
 		return
 	}
 	if user.Status == nil {
-		user.Status = &cephv1.Status{}
+		user.Status = &cephv1.ObjectStoreUserStatus{}
 	}
 
 	user.Status.Phase = status
+	user.Status.SecretName = fmt.Sprintf("rook-ceph-object-user-%s-%s", user.Spec.Store, user.Name)
```

Comment:
Please create a function to build the secret name + unit test.

Reply:
> Please create a function to build the secret name + unit test.

Done.